### PR TITLE
Add ecology files in docs/

### PR DESCRIPTION
**Pull request by coopy**
This is to support usage of [`builder-victory-component`](https://github.com/FormidableLabs/builder-victory-component) archetype.

### CODE REVIEW

File: docs/docs.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { VictoryTheme } from "formidable-landers";
+
+@Radium
+class Docs extends React.Component {
+  render() {
+    return (
+      <div>
+        <Ecology
+          overview={require("!!raw!./ecology.md")}
+          source={require("!!docgen!../src/components/victory-line")[0]}
```
Comment (boygirl) [inline docs/docs.jsx]: `victory-line` -> `victory-component-boilerplate`

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<html>
+<head>
+  <!--[if lt IE 9]>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.2/html5shiv-printshiv.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.7/es5-shim.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.7/es5-sham.min.js"></script>
+  <![endif]-->
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>VictoryLine</title>
```
Comment (boygirl) [inline docs/index.html]: `VictoryLine` -> `VictoryComponentBoilerplate`

**Comment by boygirl** [comment]
One more naming thing, then approved

**Comment by coopy** [comment]
Fixed, waiting for CI

File: docs/docs.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import { VictoryTheme } from "formidable-landers";
+
+@Radium
+class Docs extends React.Component {
+  render() {
+    return (
+      <div>
+        <Ecology
+          overview={require("!!raw!./ecology.md")}
+          source={require("!!docgen!../src/components/victory-component-boilerplate")[0]}
```
Comment (boygirl) [inline docs/docs.jsx:15]: Can you change this to the fix that doesn't involve `docgen-loader`?

Comment (coopy) [inline docs/docs.jsx:15]: Oops, will open separate PR

**Pull request merged**